package.json

```diff
@@ -79,12 +79,11 @@
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^2.0.1",
     "karma-webpack": "^2.0.5",
-    "lerna": "^3.10.5",
+    "lerna": "^4.0.0",
     "mini-css-extract-plugin": "^0.5.0",
     "mkdirp": "^0.5.1",
     "mocha": "^4.0.1",
     "ncp": "^2.0.0",
-    "node-sass": "^4.8.3",
```

Comment:
Not only is this woefully out of date, it's also deprecated entirely in favor of `sass`. This is the module that was giving me grief because it was calling `node-gyp@3.8.0` and that was throwing some errors in the `g++` compiler for...baffling reasons.